designer/server/src/test/scala/pl/touk/nussknacker/ui/api/StatisticsApiHttpServiceSecuritySpec.scala

```diff
@@ -0,0 +1,40 @@
+package pl.touk.nussknacker.ui.api
+
+import io.restassured.RestAssured.`given`
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.engine.version.BuildInfo
+import pl.touk.nussknacker.test.base.it.NuItTest
+import pl.touk.nussknacker.test.config.WithSimplifiedDesignerConfig
+import pl.touk.nussknacker.test.{NuRestAssureExtensions, NuRestAssureMatchers}
+
+class StatisticsApiHttpServiceSecuritySpec
+    extends AnyFreeSpecLike
+    with NuItTest
+    with WithSimplifiedDesignerConfig
+    with NuRestAssureExtensions
+    with NuRestAssureMatchers {
+
+  private val nuVersion = BuildInfo.version
+
+  "The statistic URL endpoint when" - {
+    "require no auth to return single URL" in {
+      given()
+        .when()
+        .noAuth()
```

Comment:
Are we sure? This potentially leads to an information leak. Maybe the frontend should only send the info where a user is logged. But the leak will be there too (are we sure that the information is not user's specific?). Maybe we should discuss this.

Reply:
I agree, I will change that